BL-469 Display desc, not value

sensei100 opened the pull request:
BL-469 Physical material type should use desc rather than value
- 991025183159703811 should display as Sound Recording, not RECORD
<img width="781" alt="screen shot 2018-06-08 at 8 53 02 am" src="https://user-images.githubusercontent.com/15167238/41159115-5f4261a4-6af9-11e8-8249-299623a49685.png">

dkinzer commented on config/environments/development.rb:
```diff
@@ -13,10 +13,10 @@
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = true
+  config.cache_classes = false
```
I think this got in by mistake.

dkinzer commented on app/helpers/alma_data_helper.rb:
```diff
@@ -34,8 +34,8 @@ def description(item)
   def physical_material_type(item)
-    if item.physical_material_type.present? && !PHYSICAL_TYPE_EXCLUSIONS.match(item.physical_material_type)
-      return item.physical_material_type
+    if item.physical_material_type.present? && !PHYSICAL_TYPE_EXCLUSIONS.match(item.physical_material_type["value"])
```
I believe this bit is can be more succinctly expressed

`if !"#{item&.physical_material_type["value"]}".match(PHYSICAL_TYPE_EXCLUSION)`

Or for readabilities sake

```
type = "#{item&.physical_material_type["value"]}"
if !type.match(PHYSICAL_TYPE_EXCLUSIONS)
```

sensei100 commented:
@dkinzer I included the wrong file in my last commit.  Sorry about that.  I reverted the config file and added the spec file that I intended to commit.  I also made the changes to the method that you requested.
